js/stat.ts

```diff
@@ -12,7 +23,10 @@ import { FileInfo, FileInfoImpl } from "./file_info";
  *       assert(fileInfo.isFile());
  */
 export async function lstat(filename: string): Promise<FileInfo> {
-  return res(await dispatch.sendAsync(...req(filename, true)));
+  const f = await res(await dispatch.sendAsync(...req(filename, true)));
+  f.path = filename;
```

Comment:
I wonder if @ry wants to keep `path` given his comment in #1770. Golang's `FileInfo` has only `name`.

Reply:
path is useful e.g. https://github.com/denoland/deno_std/issues/189